package.json

```diff
@@ -116,6 +116,11 @@
     "redux-devtools": "3.4.1",
     "redux-devtools-dock-monitor": "1.1.3",
     "redux-devtools-log-monitor": "1.4.0",
+    "redux-storage": "4.1.2",
+    "redux-storage-decorator-debounce": "1.1.3",
+    "redux-storage-decorator-filter": "1.1.8",
+    "redux-storage-decorator-migrate": "1.1.0",
+    "redux-storage-engine-electron-store": "1.1.0",
```

Comment:
seems like you "forked" `redux-storage-engine-electron-store` so there is no need for the dependency

Reply:
Yep, good catch! Initially I had added it, and then realized it didn't expose the electronStore to the window.